lib/pages/chat/chat.dart

```diff
@@ -136,13 +147,23 @@ class ChatController extends State<Chat>
       );
 
   final AutoScrollController scrollController = AutoScrollController();
+  final AutoScrollController pinnedMessageScrollController =
```

Comment:
ChatController too big
separate it into ChatPinMessageMixin

Reply:
done